packages/dashboard/src/stores/client-store.ts

```diff
@@ -3,6 +3,9 @@ import { action, observable, computed } from 'mobx';
 import { TClientProps, TCurrenciesList, TUpgradeInfo, TAccountList } from 'Types';
 import BaseStore from './base-store';
 
+type TExtendedAccount = NonNullable<TAccountList>[0] & {
```

Comment:
Here I had to extend the type for an account because in the new @deriv/api-types version 1.0.48 this type doesn't include the field `is_wallet`.